html/js/hardware.js

```diff
@@ -334,13 +334,45 @@ function HardwareManager(options) {
       form.find('input[name=max]').prop('disabled', disabled)
     }
 
-    this.toggleSensibility = function (port, select, actuators, actuatorUri) {
-      var currentActuator = actuators[actuatorUri]
-      if (currentActuator && currentActuator.steps.length === 0) {
-        select.parent().parent().hide()
-      } else if (!((port.properties.indexOf("integer") >= 0 || port.properties.indexOf("toggled") >= 0 || port.properties.indexOf("trigger") >= 0)
-        && port.symbol != ":bypass" && port.symbol != ":presets")) {
-        select.parent().parent().show()
+    this.portSupportsSensibility = function(port) {
```

Comment:
I think the word you're looking for here is Sensitivity. Although the word Sensibility exists in english, it has a different meaning (while in portuguese both translate to the same word)

Reply:
good catch. though looking at the code the `sensibility` wording is what is being used. (while the actual user-facing wording uses the correct `sensitivity` haha)
I will fix this up in this branch and merge after, thanks